test/e2e/tokens/erc20.test.mjs

```diff
@@ -529,7 +529,7 @@ describe('ERC20 tests', () => {
               fee,
             );
             await emptyL2(nf3Users[0]);
-            await new Promise(resolve => setTimeout(resolve, 15000));
+            await new Promise(resolve => setTimeout(resolve, 30000));
```

Comment:
Why increasing this timer to 30 seconds solves the issue? Doesn’t it simply reduce the possibility of it happening rather than eliminating the problem?

Reply:
I'm not sure I understand the question. The description explains why we added longer wait.

In theory, the proper way to do these tests would be to test for success from updates to balance after a transaction as suggested in #513 Ze wanted to pick this as a different issue instead of in the e2e tests PR he created.

Reply:
That was my question i guess. This is not so much a permanent fix but a workaround to minimize the probability of it happening.